packages/slice-machine/lib/models/common/widgets/Image/Form.tsx

```diff
@@ -37,6 +37,9 @@ const FormFields = {
             name: "Thumbnails",
             message: "Must set name and width or height at minimum",
             test: function (value) {
+              if (!value.name) {
```

Comment:
Wouldn't that trigger the validation right at the beginning when you open the modal ?

Reply:
nope, actually that was in the old version :)